### PR TITLE
Docs/pgd/quickstart further exploration failover

### PR DESCRIPTION
**Pull request by djw-m**

### CODE REVIEW

File: product_docs/docs/pgd/5/quickstart/index.mdx [hunk 1]
```diff
@@ -7,44 +7,57 @@ indexCards: none
...
 ## Introducing PGD and TPA
 
-PGD (Postgres Distributed) is a multi-master replicating implementation of Postgres designed for high performance and availability. The installation of PGD is orchestrated by TPA. 
+PGD is a multi-master replicating implementation of Postgres designed for high performance and availability. The installation of PGD is orchestrated by TPA. 
 
-We created TPA (Trusted Postgres Architect) to make the installation and management of various Postgres configurations easily repeatable. TPA orchestrates the creation and deployment of Postgres. 
+We created TPA to make installing and managing various Postgres configurations easily repeatable. TPA orchestrates creating and deploying Postgres. 
 
-In these quick starts, we install TPA first. If you already have TPA installed, you can skip those steps. TPA can be used, and re-used, to deploy various configurations of Postgres clusters.
+In these quick starts, you install TPA first. If you already have TPA installed, you can skip those steps. You can use TPA to deploy various configurations of Postgres clusters.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/quickstart/index.mdx:36]: I don't think we need to explicitly state that you can use a piece of software for its intended purpose more than once. If you mean something else, please elaborate. If you do need to use the word "reuse,"  it's spelled without a hyphen.

File: product_docs/docs/pgd/5/quickstart/further_explore_failover.mdx [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+```
+
+In a more readable form, that is:
+
+```shell
+while true; 
+    do psql -h kaftan,kaolin,kaboom -p 6432 bdrdb -c \
+        "INSERT INTO ping(node, timestamp) select node_name, current_timestamp from bdr.local_node_summary;"
+    done
+```
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/quickstart/further_explore_failover.mdx:181]: Why would you put it in the less-readable form?

**Review by ebgitelman — COMMENTED** [review]
Please use second person (you) for things the user will do, not first person plural (we). Reserve "we" for things actions that EDB specifically takes or recommendations (we recommend). 

I left the tone casual but reduced future tense where it was affecting clarity. Eliminated idioms that non-native English speakers might not understand. Reduced wordiness to help with clarity.

File: product_docs/docs/pgd/5/quickstart/further_explore_failover.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/quickstart/further_explore_failover.mdx:181]: So it copies and pastes into the console correctly.

File: product_docs/docs/pgd/5/quickstart/index.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/quickstart/index.mdx:36]: That was a requested emphasis, that TPA is not just an installation application for one installation, but can be used for many different installations.

File: product_docs/docs/pgd/5/quickstart/index.mdx [hunk 3]
```diff
@@ -12,11 +12,11 @@ navigation:
 
 ## Quick Start
 
-Postgres Distributed (PGD) is a multi-master replicating implementation of Postgres designed for high performance and availability. You can create database clusters comprising of many bidirectionally synchronizing database nodes. The clusters can have a number proxy servers that direct your query traffic to the most available nodes, adding further resilience to your cluster configuration.
+Postgres Distributed (PGD) is a multi-master replicating implementation of Postgres designed for high performance and availability. You can create database clusters made up of of many bidirectionally synchronizing database nodes. The clusters can have a number of proxy servers that direct your query traffic to the most available nodes, adding further resilience to your cluster configuration.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/quickstart/index.mdx:15]: "made up of of" - double of?

File: product_docs/docs/pgd/5/quickstart/index.mdx (hunk 1 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/quickstart/index.mdx:38]: I would be more explicit about what you are trying to say in that case. Like a second sentence, something like: You can also use TPA in the future or more than once to ...

File: product_docs/docs/pgd/5/quickstart/index.mdx (hunk 3 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/quickstart/index.mdx:17]: typo, good catch! Feel free to fix.

**Comment by josh-heyer** [comment]
⚠Rebased!

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**